Makefile

```diff
@@ -107,6 +107,7 @@ clean:
 	rm -f docker-compose_shared.yml
 	rm -fr ./mig
 	rm -fr ./httpd
+	rm -fr ./certs
```

Comment:
For the common case with auto-generated certs it makes sense, but I can't support that general removal as part of `clean` because we use a symlink to point `certs` to an externally maintained (LetsEncrypt) folder. I really can't be bothered with setting it up again repeatedly.
Perhaps it would work to make that particular removal conditional so that it bails out on symlinks.

Reply:
`[ -L ./certs ] || rm -rf ./certs`
appears to work here.

Reply:
Works for me! Thx